hasura/metadata/databases/default/tables/public_activities.yaml

```diff
@@ -48,11 +48,21 @@ select_permissions:
         - created_at
         - updated_at
       filter:
-        _and:
+        _or:
           - organization:
               circles:
                 users:
-                  profile:
-                    id:
+                  _and:
+                    - profile:
+                        id:
+                          _eq: X-Hasura-User-Id
+                    - deleted_at:
+                        _is_null: true
+          - organization:
+              members:
+                _and:
+                  - id:
```

Comment:
this is a bug, because that's the org_member row id, not the profile id; will fix in #1995